src/black/report.py

```diff
@@ -4,6 +4,7 @@
 from dataclasses import dataclass
 from enum import Enum
 from pathlib import Path
+import typing
```

Comment:
Prefer `from typing import Union` (but with my suggested change above you should be able to undo all changes to this file)

Reply:
Resolved in 5f85ff5.